spyder/utils/icon_manager.py

```diff
@@ -336,6 +336,8 @@
     'dependency_ok':           [('fa.check',), {'color': MAIN_FG_COLOR}],
     'dependency_warning':      [('fa.warning',), {'color': 'orange'}],
     'dependency_error':        [('fa.warning',), {'color': 'darkred'}],
+    'broken_variant':          [('mdi.image-broken-variant',),
```

Comment:
Let's call this `broken_image` instead of `broken_variant`, so the name of the icon is easier to remember.